waf

```diff
@@ -1,14 +1,14 @@
 #!/usr/bin/python
 
 from __future__ import print_function
-import os
+import subprocess
 import os.path as p
 import sys
 
 d = p.dirname(p.realpath(__file__))
 try:
-    os.execv(os.path.join(d, 'modules/waf/waf-light'), sys.argv)
-except OSError:
+    subprocess.check_call(['python', p.join(d, 'modules', 'waf', 'waf-light')] + sys.argv[1:])
+except subprocess.CalledProcessError:
     print('Missing waf submodule. Trying to get it')
```

Comment:
I think we should verify the existence of `modules/waf/waf-light` before running it.